src/katsdpcontroller/generator.py

```diff
@@ -1647,7 +1647,8 @@ def _make_ingest(
             ingest.physical_factory = IngestTask
         ingest.fake_katcp_server_cls = FakeIngestDeviceServer
         ingest.image = "katsdpingest_" + normalise_gpu_name(defaults.INGEST_GPU_NAME)
-        ingest.command = ["ingest.py"]
+        ingest.command = ["capambel", "-c", "cap_net_raw+p", "-v", "--", "ingest.py"]
```

Comment:
The `-v` shouldn't be needed for production - it just spews out debug information.